server/lib/nl/fulfillment/fulfiller.py

```diff
@@ -161,7 +169,13 @@ def _perform_strict_mode_checks(uttr: Utterance) -> bool:
 
 
 def _produce_query_types(uttr: Utterance) -> List[QueryType]:
-  query_types = [handlers.first_query_type(uttr)]
+  query_types = []
+  if params.is_bio(uttr.insight_ctx) and uttr.entities and uttr.properties:
+    query_types.append(QueryType.TRIPLE)
+  # The remaining query types require places to be set
+  if not uttr.places:
+    return query_types
```

Comment:
To confir, even if set, this is not handled in the callsite now, right?

Reply:
these other query types will only be handled if the triple handler fails AND there are both places and svs